index.html

```diff
@@ -20,15 +20,15 @@
         <h1 class="name">Hey there. I’m<br>Mariane</h1>
         <h1 class="detail">I’m a software developer</h1>
         <p class="pOne">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
```

Comment:
- [ ] **OPTIONAL:** The paragraph should have the same padding on left and right side with the logo and hamburger button, please fix it.